Bookish/Models/EditMembers.cs

```diff
@@ -0,0 +1,42 @@
+namespace Bookish.Models;
+
+public class EditMembers
+{
+    public static void InsertMembers(MemberModel newmember)
+    {
+        using var context = new LibraryContext();
+        var member = new MemberModel();
+        foreach (var property in typeof(MemberModel).GetProperties())
+        {
+            if (property.Name != "Id")
+            {
+                property.SetValue(member, property.GetValue(newmember));
+            }
+        }
+        context.Members.Add(member);

```

Comment:
Can we add the `newmember` directly without needing to make a copy?

Reply:
Yes, apparently - I think this code previously accepted separate properties and constructed the object, but now accepts an object and so constructing a new object is entirely pointless.